### PR TITLE
Fixes #22455: Documentation of plugin version compatibility is outdated

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/22455

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -53,39 +53,29 @@ Each plugin build information are grouped in file `build.conf` in plugin root di
...
+Plugins are linked to Rudder patch version, so we retrieve in `rudder-plugins` the same branch
 structure than in `rudder`. Moreover, one needs to always compile and use a plugin for the
-corresponding Rudder version:
+corresponding Rudder patch version:
 
 ```
-- master (plugin compatible with Rudder next version, i.e developing branch)
-- branches/rudder/5.0 (plugins compatible with Rudder 5.0)
-- branches/rudder/5.1 (plugins compatible with Rudder 5.1)
+- branches/rudder/7.2-next (plugin compatible with Rudder next 7?2 version, i.e developing branch)
```
Comment (amousset) [inline README.adoc:61]:
```suggestion
- branches/rudder/7.2-next (plugin compatible with Rudder next 7.2 version, i.e developing branch)
```

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**